### PR TITLE
[docs] update image integration README

### PR DESCRIPTION
**Pull request by sarah11918**
## Changes

Some suggested updates and clarifications based on Image Interrogation with @tony-sull 
- more thorough treatment of when Squoosh/sharp do/don't work
- updating some properties for remote image guidance, where defaults don't hold
- includes mention of `alt` being required
- more clearly describes what happens re: building/optimizing to images in public folder

## Testing

No tests

## Docs

only docs

### CODE REVIEW

File: packages/integrations/image/README.md [hunk 1]
```diff
@@ -22,8 +22,20 @@ This integration provides `<Image />` and `<Picture>` components as well as a ba
 
 ## Installation
 
+The default image transformer is based on [Squoosh](https://github.com/GoogleChromeLabs/squoosh) and uses web assembly libraries to support most deployment environments, including those that do not support sharp, such as StackBlitz and Netlify Edge.
```
Comment (delucis) [inline packages/integrations/image/README.md:25]: It looks kind of bad, but I believe the canonical spelling of “WebAssembly” is pascal case.

Also wondering about leading with “including those that do not support sharp” — we haven’t mentioned sharp at all at this point, so might be confusing why we care that this thing we haven’t mention is or isn’t supported. Is being more generic for a second unhelpful?

```suggestion
The default image transformer is based on [Squoosh](https://github.com/GoogleChromeLabs/squoosh) and uses WebAssembly libraries to support most deployment environments, including StackBlitz and Netlify Edge, which don’t support all image libraries.
```

Comment (delucis) [reply packages/integrations/image/README.md:25]: Or, given we move straight into discussion of sharp, flip things, so in the next section we say where sharp won’t work and here keep it to:

```suggestion
The default image transformer is based on [Squoosh](https://github.com/GoogleChromeLabs/squoosh) and uses WebAssembly libraries to support most deployment environments.
```

File: packages/integrations/image/README.md [hunk 2]
```diff
@@ -22,8 +22,20 @@ This integration provides `<Image />` and `<Picture>` components as well as a ba
...
 
- ### Quick Install
+For faster builds and more fine-grained control of image transformations, we recommend also installing [sharp](https://sharp.pixelplumbing.com/) if
+- You are building a static site with Astro.
+- You are using an SSR deployment host that supports NodeJS using `@astrojs/netlify/functions`, `@astrojs/vercel/serverless` or `@astrojs/node`.
+
```
Comment (delucis) [inline packages/integrations/image/README.md:30]: Is two “You are” and one “You are not” confusing or could this work?

```suggestion
- You are not using an environment that does not support sharp, such as StackBlitz or Netlify Edge.

```

**Review by delucis — COMMENTED** [review]
Once again, hats off to @sarah11918 for digging into this and @tony-sull for helping out!

I’ve left a couple smaller comments but on the whole this looks great and could probably be merged as-is from where I’m standing.

File: packages/integrations/image/README.md (hunk 1 above)
Comment (sarah11918) [inline packages/integrations/image/README.md:25]:
```suggestion
The default image transformer is based on [Squoosh](https://github.com/GoogleChromeLabs/squoosh) and uses WebAssembly libraries to support most deployment environments, including those that do not support sharp, such as StackBlitz and Netlify Edge.
```

File: packages/integrations/image/README.md [hunk 3]
```diff
@@ -22,8 +22,20 @@ This integration provides `<Image />` and `<Picture>` components as well as a ba
 
 ## Installation
 
+The default image transformer is based on [Squoosh](https://github.com/GoogleChromeLabs/squoosh) and uses WebAssembly libraries to support most deployment environments, including those that do not support sharp, such as StackBlitz and Netlify Edge.
```
Comment (sarah11918) [inline packages/integrations/image/README.md:25]:
```suggestion
Along with our integration, we recommending installing sharp where appropriate. The `@astrojs/image` default image transformer is based on [Squoosh](https://github.com/GoogleChromeLabs/squoosh) and uses WebAssembly libraries to support most deployment environments, including those that do not support sharp, such as StackBlitz and Netlify Edge.
```

File: packages/integrations/image/README.md [hunk 4]
```diff
@@ -22,8 +22,20 @@ This integration provides `<Image />` and `<Picture>` components as well as a ba
...
 
- ### Quick Install
+For faster builds and more fine-grained control of image transformations, we recommend also installing [sharp](https://sharp.pixelplumbing.com/) if
```
Comment (sarah11918) [inline packages/integrations/image/README.md:27]:
```suggestion
For faster builds and more fine-grained control of image transformations, install sharp if
```

File: packages/integrations/image/README.md [hunk 5]
```diff
@@ -22,8 +22,22 @@ This integration provides `<Image />` and `<Picture>` components as well as a ba
 
 ## Installation
 
+Along with our integration, we recommending installing [sharp](https://sharp.pixelplumbing.com/) where appropriate.
```
Comment (sarah11918) [inline packages/integrations/image/README.md:25]:
```suggestion
Along with our integration, we recommending installing [sharp](https://sharp.pixelplumbing.com/) when appropriate. 
```

File: packages/integrations/image/README.md [hunk 6]
```diff
@@ -22,8 +22,22 @@ This integration provides `<Image />` and `<Picture>` components as well as a ba
 
 ## Installation
 
+Along with our integration, we recommending installing [sharp](https://sharp.pixelplumbing.com/) when appropriate. 
 
- ### Quick Install
+The `@astrojs/image` default image transformer is based on [Squoosh](https://github.com/GoogleChromeLabs/squoosh) and uses WebAssembly libraries to support most deployment environments, including those that do not support sharp, such as StackBlitz and Netlify Edge.
+
+For faster builds and more fine-grained control of image transformations, install sharp if
```
Comment (sarah11918) [inline packages/integrations/image/README.md:29]:
```suggestion
For faster builds and more fine-grained control of image transformations, install sharp in addition to `@astrojs/image` if
```

File: packages/integrations/image/README.md [hunk 7]
```diff
@@ -22,8 +22,22 @@ This integration provides `<Image />` and `<Picture>` components as well as a ba
 
 ## Installation
 
+Along with our integration, we recommending installing [sharp](https://sharp.pixelplumbing.com/) when appropriate.
```
Comment (delucis) [inline packages/integrations/image/README.md:25]: Nice! Love the revision.

Just a small typo here:

```suggestion
Along with our integration, we recommend installing [sharp](https://sharp.pixelplumbing.com/) when appropriate. 
```

Comment (sarah11918) [reply packages/integrations/image/README.md]: Thanks for the help!  💜

File: packages/integrations/image/README.md [hunk 8]
```diff
@@ -22,8 +22,22 @@ This integration provides `<Image />` and `<Picture>` components as well as a ba
 
 ## Installation
 
+Along with our integration, we recommend installing [sharp](https://sharp.pixelplumbing.com/) when appropriate. 
 
- ### Quick Install
+The `@astrojs/image` default image transformer is based on [Squoosh](https://github.com/GoogleChromeLabs/squoosh) and uses WebAssembly libraries to support most deployment environments, including those that do not support sharp, such as StackBlitz and Netlify Edge.
```
Comment (tony-sull) [inline packages/integrations/image/README.md:27]: I *think* Netlify Edge runs on deno so the integration won't work there, it still depends on Node even without sharp

File: packages/integrations/image/README.md [hunk 9]
```diff
@@ -22,8 +22,22 @@ This integration provides `<Image />` and `<Picture>` components as well as a ba
...
+For faster builds and more fine-grained control of image transformations, install sharp in addition to `@astrojs/image` if
+- You are building a static site with Astro.
+- You are using an SSR deployment host that supports NodeJS using `@astrojs/netlify/functions`, `@astrojs/vercel/serverless` or `@astrojs/node`.
+
+
+Note that `@astrojs/image` is not currently supported on
+- Cloudflare SSR
+- @astrojs/deno
+- @astrojs/vercel/edge
+
```
Comment (tony-sull) [inline packages/integrations/image/README.md:38]: `@astrojs/netlify/edge-functions` probably belongs here as well since it runs on deno

**Review by tony-sull — CHANGES_REQUESTED** [review]
Looks good to me!  Marked a couple quick notes where we should list Netlify Edge as unsupported as well

File: packages/integrations/image/README.md [hunk 10]
```diff
@@ -22,8 +22,22 @@ This integration provides `<Image />` and `<Picture>` components as well as a ba
...
+For faster builds and more fine-grained control of image transformations, install sharp in addition to `@astrojs/image` if
+- You are building a static site with Astro.
+- You are using an SSR deployment host that supports NodeJS using `@astrojs/netlify/functions`, `@astrojs/vercel/serverless` or `@astrojs/node`.
+
+
+Note that `@astrojs/image` is not currently supported on
+- Cloudflare SSR
+- @astrojs/deno
+- @astrojs/vercel/edge
```
Comment (sarah11918) [inline packages/integrations/image/README.md:37]:
```suggestion
Note that `@astrojs/image` is not currently supported on
- Cloudflare SSR
- `@astrojs/deno`
- `@astrojs/netlify/edge-functions`
- `@astrojs/vercel/edge`
```

File: packages/integrations/image/README.md [hunk 11]
```diff
@@ -22,8 +22,23 @@ This integration provides `<Image />` and `<Picture>` components as well as a ba
 
 ## Installation
 
+Along with our integration, we recommend installing [sharp](https://sharp.pixelplumbing.com/) when appropriate. 
 
- ### Quick Install
+The `@astrojs/image` default image transformer is based on [Squoosh](https://github.com/GoogleChromeLabs/squoosh) and uses WebAssembly libraries to support most deployment environments, including those that do not support sharp, such as StackBlitz and Netlify Edge.
```
Comment (sarah11918) [inline packages/integrations/image/README.md:27]:
```suggestion
The `@astrojs/image` default image transformer is based on [Squoosh](https://github.com/GoogleChromeLabs/squoosh) and uses WebAssembly libraries to support most deployment environments, including those that do not support sharp, such as StackBlitz.
```

**Comment by sarah11918** [comment]
Thanks @tony-sull! How about now?

**Review by tony-sull — APPROVED** [review]
Looks great!

**Pull request merged**